connections/conn-upgrade.plantuml

```diff
@@ -1,36 +1,35 @@
 @startuml
 
 entity Initiator
-entity Responder
+entity Receiver
 
-== Initial multistream handshake ==
+== Initial protocol handshake ==
 
-Responder -> Initiator: /multistream/1.0.0
-Initiator -> Responder: /multistream/1.0.0
+Initiator -> Receiver: /multistream/1.0.0
+Receiver -> Initiator: /multistream/1.0.0
```

Comment:
We borrowed this terminology from the Noise Spec: https://noiseprotocol.org/noise.html

Reply:
To what should that be changed, then?